index.html

```diff
@@ -25,36 +25,52 @@
   <body>
     <header>
       <nav class="navbar">
-        <img
-          class="nav logo icon"
-          src="images/image_geometry_1.svg"
-          alt="logo"
-        />
-        <img class="nav ic-menu icon" src="images/ic_menu.svg" alt="ic-menu" />
+        <img class="icon" src="images/image_geometry_1.svg" alt="logo" />
+        <img class="ic-menu icon" src="images/ic_menu.svg" alt="ic-menu" />
+        <ul class="nav-list">
+          <a href="#hello"> <li>Hello</li></a>
+          <a href="#projects"> <li>Portfolio</li></a>
+          <a href="#about"> <li>About</li></a>
+          <a href="#contact"> <li>Contact</li></a>
+        </ul>
       </nav>
     </header>
     <main>
-      <section class="white-bg">
-        <h1 class="heading">Charlie Giorgania</h1>
-        <p class="para">
-          Hello! I am a software developer! I can help you build a product,
-          feature or website. Taka a look of my works. If you like what you see
-          and have a project you need coded, don’t hesitate and contact me.
-        </p>
-        <button class="buttons-primary-dark" type="button">
-          Start collaboration
-        </button>
+      <section class="intro-section" id="hello">
+        <article class="white-bg">
+          <h1 class="heading">Charlie Giorgania</h1>
+          <p class="para">
+            Hello! I am a software developer! I can help you build a product,
+            feature or website. Taka a look of my works. If you like what you
+            see and have a project you need coded, don’t hesitate and contact
+            me.
+          </p>
+          <button class="buttons-primary-dark" type="button">
+            Start collaboration
+          </button>
+        </article>
+        <img
+          class="page-brk-dp"
+          src="images/image-geometry_1.svg"
+          alt="page break"
+        />
       </section>
```

Comment:
- [ ] You have properly positions the elements in this section, however, I observed that the padding added is less than the original design. Let's fix the padding to look more like the template.

Your Implementation | Figma Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/149514191-71d1ef5d-4e30-4d48-bde4-9bf82aaf557e.png) | ![image](https://user-images.githubusercontent.com/27709832/149514218-90550aab-f15a-45fe-acec-9440e6c63db6.png)